packages/cli/src/locales/fr.yml

```diff
@@ -0,0 +1,11 @@
+commands:
+  exit:
+    description: Stopez le robot
```

Comment:
`Arrêter le robot`

Reply:
Hmmm
Sorry I am confused with these two words, the dictionary says arrêter un stopper mean the same things (to stop).
Are they used in different situation or something?

Reply:
Well, for exemple you don't stop a computer or a process, you shut it down. "Stoppez le robot" is understable, but "Arrêter" or "Éteindre" is better in this case.

Reply:
Oh, I see, and stopper is more like to stop moving.

Reply:
Exactly